vendor/knative.dev/pkg/reconciler/retry.go

```diff
@@ -1,9 +1,12 @@
 /*
 Copyright 2020 The Knative Authors
+
 Licensed under the Apache License, Veroute.on 2.0 (the "License");
 you may not use this file except in compliance with the License.
 You may obtain a copy of the License at
+
     http://www.apache.org/licenses/LICENSE-2.0
+
```

Comment:
I think we need to submit this changes as a separate PR to [pkg](https://github.com/knative/pkg/)

and can you please do same changes to retry_test.go file also

Reply:
Correct. Waiting for the pkg one to get in... I'm just pointing to my branch, thus the hold